docs/dev/run_the_etl.rst

```diff
@@ -178,33 +178,33 @@ Add ``DAGSTER_HOME`` to the currecnt session with
 
     $ export DAGSTER_HOME=/path/to/dagster_home
 
-Once ``DAGSTER_HOME`` is set, launch Dagit by running:
+Once ``DAGSTER_HOME`` is set, launch the dagster UI by running:
 
 .. code-block:: console
 
-    $ dagit -m pudl.etl -m pudl.ferc_to_sqlite
+    $ dagster-webserver -m pudl.etl -m pudl.ferc_to_sqlite
 
-To avoid typing out the dagit command each time you want to launch it,
-you can create an alias for the command in your shell:
+To avoid typing out the ``dagster-webserver`` command each time you want to launch
+the UI, you can create an alias for the command in your shell:
 
 .. code-block:: console
 
-    $ echo "alias launch_dagit='dagit -m pudl.etl -m pudl.ferc_to_sqlite'" >> ~/.zshrc # zsh
-    $ echo "alias launch_dagit='dagit -m pudl.etl -m pudl.ferc_to_sqlite'" >> ~/.bashrc # bash
-    $ alias launch_dagit="dagit -m pudl.etl -m pudl.ferc_to_sqlite" # fish
+    $ echo "alias launch_dagster='dagster-webserver -m pudl.etl -m pudl.ferc_to_sqlite'" >> ~/.zshrc # zsh
+    $ echo "alias launch_dagster='dagster-webserver -m pudl.etl -m pudl.ferc_to_sqlite'" >> ~/.bashrc # bash
+    $ alias launch_dagster="dagster-webserver -m pudl.etl -m pudl.ferc_to_sqlite" # fish
```

Comment:
Similarly does it make more sense to tell folks to use `dagster-webserver` or `dagster dev` here?